data/kamailio.cfg

```diff
@@ -768,8 +984,20 @@ route[CHECK_STATS] {
 		$var(stats_data) = $var(stats_data) + " {\"index\": {\"_index\":\"homer-"+ $timef(%d.%m.%Y) +"\",\"_type\":\""+$var(type)+"\" }}\n";
 		$var(stats_data) = $var(stats_data) + $var(tmp_json) + "\n";
 
+		
+		#!ifdef TO_GRAYLOG
+		$var(short) = @cfg_get.short.message;
+		$var(full) = @cfg_get.full.message;
+		jansson_set("string", "short_message", ""+$var(short)+"", "$var(tmp_json)");
+		jansson_set("string", "full_message", ""+$var(full)+"", "$var(tmp_json)");
+		$var(graylog_json) = $var(tmp_json);
+		#!endif
+		
 		$var(tmp_json) = 0;
 		$sht(a=>packet::count) = 0;
+
+		$var(body) = $var(stats_data);
+		route(SEND_ES_DATA);
```

Comment:
Why did you split data sending in this way. In this way you are basically sending an HTTP request per document (elasticsearch or graylog), loosing the benefit of the _bulk API. For this reason the stats where grouped by type.